components/backup/src/lib.rs

```diff
@@ -3,12 +3,8 @@
 #![recursion_limit = "200"]
 #![feature(box_patterns)]
 
-#[macro_use]
-extern crate failure;
 #[allow(unused_extern_crates)]
 extern crate tikv_alloc;
```

Comment:
Is this really used? Not sure If I can just remove it.

Reply:
You can try to remove it and build again to see whether it's necessary.

Reply:
Any update on this problem?

Reply:
Well I found this https://github.com/tikv/tikv/blob/2a2fa03da53b63f3fc24d7ea53aead40176979b5/components/tikv_alloc/src/lib.rs#L10-L12

but I'm not sure this is still true

Reply:
Well, I think we better leave it unchanged.